### ISSUE TITLE
Accessibility/Display Options 

### ISSUE THREAD

**Issue by Missive00** [Clayton-TV/claytontv#23]
# User story
I am a user who requires larger text so to be able to integrate with the site.
I am a user who requires larger icons to be able to access the site.
I am a user who would appreciate a light/dark mode to access the site.
I am a user who would like to be able to toggle between lists, tiles, and large tiles when browsing videos.

# Details

Clayton TV should have a number of simple accessibility and display options to make accessing and browsing the site more straightforward.

**Changing text size:** Either through a slider/button or interacting well with the browsers zoom setting the text size should increase/decrease.
**Changing icon size:** Either through a slider/button or interacting well with the browsers zoom setting the icon size should increase/decrease.
**Changing dark/light mode:** There should be a toggle for dark/light mode that should follow the browser/system settings by default.

**Search results can be display settings:** Search results should be visible either:

-  as tiles i.e showing the video icon, title, _?tags maybe?_ _?runtime maybe?_
- as large tiles, same as the above but with larger tiles.
- as a list same as the above but with the tiles stretching fully across the page.
- as a compact list same as the above but showing more results per page (icon could be smaller)

# Acceptance criteria
- [ ] Decide what accessibility/display options are required/should be implemented. 
- [ ] It is possible to change the text size.
- [ ] It is possible to change the icon size.
- [ ] It is possible to change the light/dark size.
- [ ] Decide how search results are displayed #7 
- [ ] Search results can be displayed either as large tiles/small tiles/list/compact list.

# Estimation of work

*Approximately how many hours/days do you expect this work to take?*

# Updates
*Any updates from further down the thread are encouraged so people can easily see the status of this issue*
